Pinta/MainWindow.cs

```diff
@@ -461,9 +458,13 @@ private void CreateDockAndPads (HBox container)
 
 		#region User Settings
 		private const string LastDialogDirSettingKey = "last-dialog-directory";
+		private const string LastSelectedToolSettingKey = "last-selected-tool";
 
 		private void LoadUserSettings ()
 		{
+			// Set selected tool to last selected or default to the PaintBrush
```

Comment:
If the value from the settings is a tool name that's no longer valid (e.g. an add-in that was removed), will this end up leaving no selected tool?

Reply:
No, it will be the first tool added to `ToolManager`.  If a tool is added to `ToolManager` and no tool is currently set, it becomes the selected tool.

After the Tools have been populated, this attempts to change the selected tool, but if it fails the original tool will remain selected.

Reply:
Thanks, that sounds good!